source/ranges.tex

```diff
@@ -5809,11 +5810,11 @@
 \begin{itemize}
 \item If \exposid{ref-is-glvalue} is \tcode{true},
   \exposid{Base} models \libconcept{bidirectional_range}, and
-  \tcode{range_reference_t<\exposid{Base}>} models
-  both \libconcept{bidirectional_range} and \libconcept{common_range},
+  \tcode{\exposid{InnerBase}} models
+  both \libconcept{bidire\-ctional_range} and \libconcept{common_range},
```

Comment:
It this the correct place to break? Shouldn't it be after the "c"?

Reply:
Yep, the check will fail if after "c".

Reply:
We don't violate English hyphenation rules. Try breaking even earlier.

Reply:
@jensmaurer Is that {bidi\-rectional_range} acceptable?